README.md

```diff
@@ -173,11 +173,11 @@ Your email will be acknowledged within 24 hours, and you’ll receive a more
 detailed response to your email within 48 hours indicating the next steps in
 handling your report.
 
-There is not any hard and fast rule to determine if a bug is worth reporting.
-The general rule is any attack worth reporting via the security address must
-allow an attacker to affect the confidentiality, integrity and availability of
-the Node.js application or its system for which the attacker does not already
-have the capability.
+There is not any hard and fast rule to determine if a bug is worth reporting as
+a security issue. The general rule is any issue worth reporting via the
+security address must allow an attacker to affect the confidentiality, integrity
+and availability of the Node.js application or its system for which the attacker
```

Comment:
I think its accurate as is. Its a security issue if it affect the app OR the system the app is on (either one). Its not necessary that it effect the app AND the system.